src/helper/sanitize.ts

```diff
@@ -139,6 +140,27 @@ export const cleanHtml = (dirty: string): string => {
     allowedTags: [ ...AllowedTags ],
     allowedAttributes: {
       '*': [ ...AllowedAttributes ],
+      input: [ 'type', 'disabled', 'checked' ]
     },
+    transformTags: {
```

Comment:
Why is this required, and why only for disabled checkboxes?

Reply:
We don't want to enable all kind of input elements through customRenderer which is being passed through this sanitizer. But for giving the option like the todo lists in markdown (which you can also see at the description of this PR) we want to enable checkboxes, if they are in disabled state. To do that we need to use the transformer to check if it matches with the expected attributes. Otherwise we're returning empty. Which seems like the only option in the [sanitize-html documentation](https://www.npmjs.com/package/sanitize-html).